From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vin=C3=ADcius=20Ferr=C3=A3o?= <2031761+viniciusferrao@users.noreply.github.com>
Date: Sun, 18 Oct 2026 10:00:00 -0300
Subject: [PATCH 23/23] Mark the xCAT build in the version string

xCAT starts the sender, and adds the receiver to an initial ramdisk,
only when the binary carries the fixes of this series. Add "(xCAT
build 1)" to the version string, which each program logs when it
starts, so that xCAT can tell this build from other UFTP 5 builds.
This patch is for the xCAT package only.
---
 uftp.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/uftp.h b/uftp.h
index 81de50d..c4cb827 100644
--- a/uftp.h
+++ b/uftp.h
@@ -157,7 +157,7 @@ typedef pthread_t thread_t;
 
 #endif // if WINDOWS
 
-#define VERSIONSTR "UFTP version 5.0.4  Copyright (C) 2001-2026  Dennis A. Bush"
+#define VERSIONSTR "UFTP version 5.0.4 (xCAT build 1)  Copyright (C) 2001-2026  Dennis A. Bush"
 #define UFTP_VER_NUM 0x50
 #define UFTP4_VER_NUM 0x40
 
